generators/ionic/resources/oauth2/cypress/support/oauth2.ts

```diff
@@ -0,0 +1,179 @@
+/* eslint-disable @typescript-eslint/no-namespace, @typescript-eslint/naming-convention */
+// eslint-disable-next-line spaced-comment
+/// <reference types="cypress" />
+import { apiHost } from './config';
+import { environment } from '../../src/environments/environment';
+
+const {
+  oidcConfig: { redirect_url: redirect_uri, scopes: scope, audience },
+} = environment;
+
+// Get oauth2 basic data
+const getOauth2Data = () =>
+  cy
+    .request({
+      url: `${apiHost}/api/auth-info`,
+      followRedirect: false,
+    })
+    .then(({ body: info }) => {
+      const { issuer, clientId } = info;
+      return cy
+        .request({
+          url: `${issuer}/.well-known/openid-configuration`,
```

Comment:
```suggestion
          url: `${(issuer.endsWith('/')) ? issuer.substring(0, issuer.length - 1) : issuer}/.well-known/openid-configuration`,
```

We have similar logic in `src/app/auth/auth-config.service.ts`:

```
if (this.authConfig.issuer.endsWith('/')) {
  this.authConfig.issuer = this.authConfig.issuer.substring(0, this.authConfig.issuer.length - 1);
}
```

Reply:
Should be fixed.